src/test/java/com/klarna/hiverunner/OrcSnappyTest.java

```diff
@@ -56,20 +55,20 @@ public void verifyThatDataIsAvailableInOrcSnappy() {
         Assertions.assertEquals(expected, actual);
     }
 
-    @Disabled // Fails with java.lang.UnsatisfiedLinkError: org.apache.hadoop.util.NativeCodeLoader.buildSupportsSnappy()Z
-    @Test
-    public void testCountOrcNocomp() {
-        List<String> expected = Arrays.asList("3");
-        List<String> actual = hiveShell.executeQuery("select count(*) from foo_orc_nocomp");
-        Assertions.assertEquals(expected, actual);
-    }
-
-    @Disabled // Fails with java.lang.UnsatisfiedLinkError: org.apache.hadoop.util.NativeCodeLoader.buildSupportsSnappy()Z
-    @Test
-    public void testCountOrcSnappy() {
-        List<String> expected = Arrays.asList("3");
-        List<String> actual = hiveShell.executeQuery("select count(*) from foo_orc_snappy");
-        Assertions.assertEquals(expected, actual);
-    }
+//    @Disabled // Fails with java.lang.UnsatisfiedLinkError: org.apache.hadoop.util.NativeCodeLoader.buildSupportsSnappy()Z
```

Comment:
If they're disabled we can just keep them? Or just delete them entirely if we think this will never work.

Reply:
Fixed. Works now